.github/workflows/run_cell-type-ewings.yaml

```
@@ -0,0 +1,68 @@
+# This is a workflow to run the cell-type-ewings module
+# Analysis modules are run based on three triggers:
+# - Manual trigger
+# - On pull requests where code in the module has changed
+# - As a reusable workflow called from a separate workflow which periodically runs all modules
+
+name: Run cell-type-ewings analysis module
+env:
+  MODULE_PATH: analyses/cell-type-ewings
+  AWS_DEFAULT_REGION: us-east-1
+
+concurrency:
+  # only one run per branch at a time
+  group: cell-type-ewings-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - analyses/cell-type-ewings/**
+jobs:
+  run-module:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Download test data
+        run: ./download-data.py --test-data --format "sce,anndata" --samples SCPCS000490
+
+      - name: Set up R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: 4.4.0
+          use-public-rspm: true
+
+      - name: Set up pandoc
+        uses: r-lib/actions/setup-pandoc@v2
+
+      - name: Install additional system dependencies
+        run: |
+          sudo apt-get install -y libcurl4-openssl-dev libglpk40 jags
+
+      - name: Set up renv
+        uses: r-lib/actions/setup-renv@v2
+        with:
+          working-directory: ${{ env.MODULE_PATH }}
+
+      - name: Set up conda
+        uses: conda-incubator/setup-miniconda@v3
+        with:
+          auto-activate-base: true
+          activate-environment: ""
+
+      - name: Install and activate locked conda environment
+        run: |
+          conda install conda-lock
+          conda-lock install --name test ${MODULE_PATH}/conda-lock.yml
```

My guess about what is going in is some kind of incomaptibility with `basilisk` or another R package.  Two solutions I can think of would be to move the conda install before the R install, or what I am suggesting here: explicitly activating the `test` environment. `setup-miniconda` is supposed to do that, but seems to be failing for some reason.

```suggestion
      - name: Set up conda
        uses: conda-incubator/setup-miniconda@v3

      - name: Install and activate locked conda environment
        run: |
          conda activate test
          conda install conda-lock
          conda-lock install --name test ${MODULE_PATH}/conda-lock.yml
```

---

Oh, never mind, I remembered the subtle thing that has to happen if you are using conda: setting the shell to `bash`: Removing my suggestion above, and reverting to what you had first, then adding the line you need:
```suggestion
      - name: Set up conda
        uses: conda-incubator/setup-miniconda@v3

      - name: Install and activate locked conda environment
        run: |
          conda install conda-lock
          conda-lock install --name test ${MODULE_PATH}/conda-lock.yml
```